common/notification/actions.ts

```diff
@@ -522,7 +522,42 @@ const _notificationActions = {
         description: 'User added a match ad-hoc meeting',
         sampleContext: { student: sampleUser, appointmentId: '1', appointment: { url: '/video-chat/1/match' } },
     },
-
+    pupil_match_appointment_reminder: {
+        description: 'Remind pupil of upcoming match appointment',
```

Comment:
I would prefer to semantically decouple the reminder (the notification) from the action, so rather name it "appointment_starts" as that's what the action signifies. Then it also makes more sense to add an action "1 hour before 'appontment starts'"